package-lock.json

```diff
@@ -4346,101 +4346,6 @@
         }
       }
     },
-    "@vue/compiler-core": {
-      "version": "3.0.11",
-      "resolved": "https://registry.npmjs.org/@vue/compiler-core/-/compiler-core-3.0.11.tgz",
-      "integrity": "sha512-6sFj6TBac1y2cWCvYCA8YzHJEbsVkX7zdRs/3yK/n1ilvRqcn983XvpBbnN3v4mZ1UiQycTvOiajJmOgN9EVgw==",
-      "dev": true,
-      "requires": {
-        "@babel/parser": "^7.12.0",
-        "@babel/types": "^7.12.0",
-        "@vue/shared": "3.0.11",
-        "estree-walker": "^2.0.1",
-        "source-map": "^0.6.1"
-      },
```

Comment:
missing vue

Reply:
@sy-records 
gotcha, I missed it.
sry, I will fix it later.